cosmo_tester/test_suites/test_snapshots/snapshots_multiple_managers_test.py

```diff
@@ -0,0 +1,135 @@
+import random
+import string
+import tempfile
+import os
+
+from cloudify_rest_client.exceptions import CloudifyClientError
+
+from cosmo_tester.framework.cfy_helper import CfyHelper
+from cosmo_tester.framework.util import create_rest_client
+from cosmo_tester.test_suites.test_blueprints.nodecellar_test import (
+    OpenStackNodeCellarTest)
+
+
+class TwoManagersTest(OpenStackNodeCellarTest):
+    """
+    This test bootstraps managers, installs nodecellar using the first manager,
+    checks whether it has been installed correctly, creates a snapshot,
+    downloads it, uploads it to the second manager, uninstalls nodecellar using
+    the second manager, checks whether nodecellar is not running indeed and
+    tears down those managers.
+
+    It is required that there is one additional manager defined in the
+    handler configuration.
+    """
+
+    def setUp(self):
+        super(TwoManagersTest, self).setUp()
+        configuration = self.env.handler_configuration
+        self.workdir2 = tempfile.mkdtemp(prefix='cloudify-testenv-')
+
+        if 'additional_managers_ips' in self.env.handler_configuration:
```

Comment:
So basically I shouldn't touch the manager which is available through self.cfy and self.client and just bootstrap two another managers?